GCRCatalogs/redmapper.py

```diff
@@ -48,11 +49,13 @@ def _subclass_init(self, catalog_root_dir,
             if k not in cosmo_astropy_allowed:
                 setattr(self.cosmology, k, v)
 
+        self.lightcone = kwargs.get('lightcone')
```

Comment:
Where does `lightcone` come from?  Is this required?

Reply:
`lightcone` is an optional (but recommended) attribute to indicate if the catalog is a light cone or a box.

Reply:
So should this be `self.lightcone = kwargs.get('lightcone', True)` to default to `True` rather than `None`?

Reply:
Hmm, I was thinking that if the `lightcone` wasn't specified in the config, the user would get `None` to indicate `lightcone` wasn't set. But setting default to `True` is also fine (since we know in this case the catalog is most likely a light cone). I'll let you decide.